src/Application/BaseApplication.php

```diff
@@ -115,7 +115,7 @@ public function registerTasks(Workflow $workflow, Deployment $deployment): void
             ->addTask(SymlinkReleaseTask::class, SimpleWorkflowStage::STEP_09_SWITCH, $this)
             ->addTask(CleanupReleasesTask::class, SimpleWorkflowStage::STEP_10_CLEANUP, $this);
 
-        if ($this->hasOption('lockDeployment') && $this->getOption('lockDeployment') === true) {
+        if ($this->provideBoolOption('lockDeployment')) {
```

Comment:
does this make sure it is also true and not false?

Reply:
Yes. Sure. It provides the value as a boolean type. Could be either false or true.

Reply:
Yes but in this case the value needs to be true which is not checked any more, or?

Reply:
$this->provideBoolOption('lockDeployment') means true, right?